src/getThreadList.js

```diff
@@ -4,22 +4,28 @@ var utils = require("../utils");
 var log = require("npmlog");
 
 module.exports = function(defaultFuncs, api, ctx) {
-  return function getThreadList(start, end, callback) {
+  return function getThreadList(start, end, pending, callback) {
```

Comment:
@RobinHoutevelts -> yup, that works (but use `utils.getType(pending) === 'Function'`) and don't do `!callback` but do `utils.getType(callback) === 'Undefined'`